metadata/src/main/java/org/apache/kafka/controller/QuorumController.java

```diff
@@ -755,11 +766,22 @@ public void handleLeaderChange(LeaderAndEpoch newLeader) {
                             newEpoch + ", but we never renounced controller epoch " +
                             curEpoch);
                     }
-                    log.warn("Becoming active at controller epoch {}.", newEpoch);
+                    log.info(
```

Comment:
This can fit in two lines... let's try to avoid "exploded" function calls that look like function bodies.